<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DeleteMessagesDialog</class>
 <widget class="QDialog" name="DeleteMessagesDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Delete old messages</string>
  </property>
  <property name="sizeGripEnabled">
   <bool>true</bool>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tabByAge">
      <attribute name="title">
       <string>By Age</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <widget class="QLabel" name="lblByAgeDeleteMessages">
         <property name="text">
          <string>Delete messages</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="gboxByAgeAge">
         <property name="title">
          <string/>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout">
          <item>
           <widget class="QRadioButton" name="rbtnByAgeOlderThan">
            <property name="text">
             <string>older than</string>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="rbtnByAgeNewerThan">
            <property name="text">
             <string>newer than</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="gboxByAgeTime">
         <property name="title">
          <string/>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QSpinBox" name="sboxByAgeTime">
            <property name="maximum">
             <number>2147483647</number>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="cboxByAgeTimeUnit">
            <property name="currentText">
             <string>minutes</string>
            </property>
            <item>
             <property name="text">
              <string>minutes</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>hours</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>days</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>weeks</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>months</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>years</string>
             </property>
            </item>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tabByDate">
      <attribute name="title">
       <string>By Date</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_5">
       <item>
        <widget class="QLabel" name="lblByDateDeleteMessages">
         <property name="text">
          <string>Delete messages</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="gboxByDateAge">
         <property name="title">
          <string/>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_4">
          <item>
           <widget class="QRadioButton" name="rbtnByDateOlderThan">
            <property name="text">
             <string>older than</string>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="rbtnByDateNewerThan">
            <property name="text">
             <string>newer than</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QDateTimeEdit" name="dateTimeEdtByDate">
         <property name="calendarPopup">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tabByCount">
      <attribute name="title">
       <string>By Count</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_7">
       <item>
        <widget class="QLabel" name="lblByCountDeleteThe">
         <property name="text">
          <string>Delete the</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="gboxByCountAge">
         <property name="title">
          <string/>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_6">
          <item>
           <widget class="QRadioButton" name="rbtnByCountOldest">
            <property name="text">
             <string>oldest</string>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="rbtnByCountNewest">
            <property name="text">
             <string>newest</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="sboxByCount">
         <property name="minimum">
          <number>1</number>
         </property>
         <property name="maximum">
          <number>2147483647</number>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="lblByCountMessages">
         <property name="text">
          <string>messages.</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>DeleteMessagesDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>DeleteMessagesDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
